addons/project/models/project.py

```diff
@@ -901,18 +912,19 @@ def _get_stat_buttons(self):
                 'show': True,
                 'sequence': 60,
             })
-            buttons.append({
-                'icon': 'users',
-                'text': _lt('Collaborators'),
-                'number': self.collaborator_count,
-                'action_type': 'action',
-                'action': 'project.project_collaborator_action',
-                'additional_context': json.dumps({
-                    'active_id': self.id,
-                }),
-                'show': True,
-                'sequence': 66,
-            })
+            if self.privacy_visibility == "portal":
+                buttons.append({
+                    'icon': 'users',
+                    'text': _lt('Collaborators'),
+                    'number': self.collaborator_count,
+                    'action_type': 'action',
+                    'action': 'project.project_collaborator_action',
+                    'additional_context': json.dumps({
+                        'active_id': self.id,
+                    }),
+                    'show': True,
+                    'sequence': 66,
+                })
```

Comment:
You could just change the condition in `show` key. 🙂 
```suggestion
            buttons.append({
                'icon': 'users',
                'text': _lt('Collaborators'),
                'number': self.collaborator_count,
                'action_type': 'action',
                'action': 'project.project_collaborator_action',
                'additional_context': json.dumps({
                    'active_id': self.id,
                }),
                'show': self.privacy_visibility == "portal",
                'sequence': 66,
            })
```